From 3a4f0aad78b105623b608ebe0806681f9b4a80fe Mon Sep 17 00:00:00 2001
From: Octavian Purdila <octavian.purdila@nxp.com>
Date: Thu, 30 Mar 2017 13:13:31 +0300
Subject: [PATCH 1630/5242] MLK-14450 ARM: dts: imx6ul-14x14-evk: remove
 duplicate i2c node

commit  25dd742340f075e0aeaf349415a179c38635afaf from
https://source.codeaurora.org/external/imx/linux-imx.git

Commit 2c2a56059ff7e7e4 ("MLK-11407-8: ARM: dts: i.mx6sx/i.mx6ul: add
ldo-bypass support") was backported from imx_4.1.y and added a
duplicate i2c node which was already present upstream.

This patch removes the duplicated node and moves the difference
(clocks for wm8960) to the existing upstream node.

Signed-off-by: Octavian Purdila <octavian.purdila@nxp.com>
Reviewed-by: Daniel Baluta <daniel.baluta@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ul-14x14-evk.dtsi |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi b/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi
index 79db0d7..9c8e91d 100644
--- a/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi
+++ b/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi
@@ -164,6 +164,8 @@
 		compatible = "wlf,wm8960";
 		reg = <0x1a>;
 		wlf,shared-lrclk;
+		clocks = <&clks IMX6UL_CLK_SAI2>;
+		clock-names = "mclk";
 	};
 
 	ov5640: ov5640@3c {
-- 
1.7.9.5

